pyproject.toml

```diff
@@ -42,6 +42,7 @@ dependencies = [
   "pip >= 22.2",
   # indirect dependencies
   "setuptools", # typically needed when pip-tools invokes setup.py
+  "toml >= 0.10.1",
```

Comment:
This line is out of place, the section says “indirect deps” but it's actually used directly now.

Though, this is not the only problem here: it should also be conditional — stdlib features a toml parser since Python 3.11 so it doesn't need an extra dep to work.

Reply:
Marking this thread as resolved.